bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/BigtableFilterAdapter.java

```diff
@@ -26,10 +29,10 @@
 public class BigtableFilterAdapter extends TypedFilterAdapterBase<BigtableFilter> {
 
   @Override
-  public RowFilter adapt(FilterAdapterContext context, BigtableFilter filter) throws IOException {
-    return filter.getRowFilter();
+  public Filter adapt(FilterAdapterContext context, BigtableFilter filter) throws IOException {
+    return FILTERS.fromProto(filter.getRowFilter());
```

Comment:
Here, I am adapting using `fromProto`, as `BigtableFilter` returns `RowFilter`. 
We could update `BigtableFilter.java` to return `com.google.cloud.bigtable.data.v2.models.Filters.Filter`

Reply:
`BigtableFilter` should probably have a `com.google.cloud.bigtable.data.v2.models.Filters.Filter` member variable.  That would likely mean deprecating the current constructor that uses `com.google.cloud.bigtable.data.v2.wrappers.Filters.Filter`, and use a `com.google.cloud.bigtable.data.v2.models.Filters.Filter` in a new constructor.  Also, please add a `BigtableFilter.getFilter()` which returns the `models.Filters.Filter` and use that getter here.

Reply:
Sure, I will add that here, again Thanks for feedback!!